### DISCUSSION TITLE
在<van-tabs/>组件中加入sticky和swipeable这俩属性后，在内容里再套<van-sticky/>吸顶组件会不生效是为什么？

### DISCUSSION THREAD

**Discussion by shuaidaia** [youzan/vant#12877] [General]
<van-tabs v-model:active="tab" @click-tab="onTabChange" :offset-top="46" swipeable sticky shrink>
            <van-tab name="transaction" :title="t('app.label_transaction')">
                        <van-sticky @scroll="handleScroll" :offset-top="88">
                            <div class="tabs-container" :class="{ 'bar-sticky': isScroll, 'bar-sticky-bottom': isScroll }">
                            </div>
                        </van-sticky>
            </van-tab>
......
像这样，这是什么原因？

**Comment by pany-ang** [comment]
https://github.com/youzan/vant/pull/12938
